rootfs/ingest/python/qserv/metadata.py

```diff
@@ -41,17 +41,51 @@
 from .http import json_get
 from .loadbalancerurl import LoadBalancerAlgorithm, LoadBalancedURL
 
+CSV = 'csv'
+TSV = 'tsv'
+TXT = 'txt'
+EXT_LIST: List[str] = [CSV, TSV, TXT]
+"""List of supported input data file extensions
+"""
+
 # ---------------------------------
 # Local non-exported definitions --
 # ---------------------------------
-_CHUNKS = "chunks"
-_FILES = "files"
-_METADATA_FILENAME = "metadata.json"
-_OVERLAPS = "overlaps"
-
+_CHUNKS: str = "chunks"
+_FILES: str = "files"
+_METADATA_FILENAME: str = "metadata.json"
+_OVERLAPS: str = "overlaps"
 _LOG = logging.getLogger(__name__)
 
 
+class FileFormat(object):
```

Comment:
`(object)` is not needed in Python3.

You can probably use `@dataclass` for class which is a collection of data members.

Reply:
:+1: